keel-core/src/main/kotlin/com/netflix/spinnaker/keel/telemetry/TelemetryEvent.kt

```diff
@@ -11,6 +11,8 @@ data class ResourceCheckSkipped(
   val skipper: String = "unknown"
 ) : TelemetryEvent()
 
+object ResourceCheckCompleted : TelemetryEvent()
```

Comment:
I wonder if `ResourceCheckCompleted` belongs here? is this a `TelemetryEvent` or just an object? if its part of the `TelemetryEvent`, it might be `data class` and not object, and it might need to hold more data, like resource info?

Reply:
Good call: I'm going to move it to the file that contains other objects that are used as events but don't have any members.

Reply:
Addressed in 5ad23592